src/oceans/components/common/Guide.jsx

```diff
@@ -98,6 +108,20 @@ let UnwrappedGuide = class Guide extends React.Component {
                       {currentGuide.textFn(getState())}
                     </div>
                   </div>
+                  {renderClickToContinueReminder && (
+                    <div style={styles.guideClickToContinueReminderContainer}>
+                      <img
+                        src={fingerClickIcon1}
+                        alt=""
```

Comment:
fill these out for accessibility?

Reply:
ooh good q -- I had a (maybe not perfect) alt text in here at one point, did you intentionally remove @breville (possibly considered decorative?) Or is it that you can't actually dismiss these via keyboard currently 😬 , which might be the bigger accessibility issue?

Reply:
I also noticed that you can tab nav between the buttons in the progression, but there's no visual indicator of what has focus. I think it's[ a result of `outline: none`,](https://github.com/code-dot-org/ml-activities/blob/bfd6e0c7df4a0b53f0550cfa91f818ff12c3915c/src/oceans/styles/index.js#L24) which apparently isn't recommended:

_> Assigning outline a value of 0 or none will remove the browser's default focus style. If an element can be interacted with it must have a visible focus indicator. Provide obvious focus styling if the default focus style is removed._

https://developer.mozilla.org/en-US/docs/Web/CSS/outline#accessibility_concerns

Reply:
Good catch.  I did make it blank intentionally in my branch, but meant to follow up.  I'm curious to know when alternate text on this particular pair of images helps with accessibility.  In other accessibility work I've done, I've focused on making sure the screenreader works when tab-navigating a UI, but it's a fairly limited experience.

It would be great to do a broader accessibility pass for the tutorial separately.

Reply:
I think given the current poor accessibility (ie, you can't actually "click to continue" with only a keyboard) treating these images as decorative (ie, alt="") is appropriate. If we make the tutorial keyboard nav friendly, then having actual alt text would make sense.

I'm taking a quick look at the broader accessibility issues while we're in here.

Reply:
Makes sense!  & doesn't need to block the PR by any means.